common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -75,7 +75,8 @@ object Mappings {
     "subLocation" -> standardAnalysedString,
     "city" -> standardAnalysedString,
     "state" -> standardAnalysedString,
-    "country" -> standardAnalysedString
+    "country" -> standardAnalysedString,
+    "englishAnalysedCatchAll" -> sStemmerAnalysedString
```

Comment:
I'm inclined to say bosh most of it in there. 
[Here's what we have at the moment](https://github.com/guardian/grid/blob/cb97663639890dd21ff940154494ea8795850194/media-api/app/lib/elasticsearch/ElasticSearch.scala#L67-L69).

I see we put the labels in there - which I think would be good too, we'd just need to make sure that happens when we [update the labels (userMetadata)](https://github.com/guardian/grid/blob/cb97663639890dd21ff940154494ea8795850194/thrall/app/lib/ElasticSearch.scala#L120).